TestVectorsAwsCryptographicMaterialProviders/dafny/TestVectorsAwsCryptographicMaterialProviders/src/TestVectors.dfy

```diff
@@ -35,11 +35,6 @@ module {:options "-functionSyntax:4"} TestVectors {
     ensures test.cmm.ValidState()
     ensures testResult && !test.vector.NegativeEncryptKeyringVector? ==> materials.Some?
   {
-    print "\nTEST===> ",
-          test.vector.name,
-          if test.vector.description.Some? then "\n" + test.vector.description.value else "",
-          if test.vector.NegativeEncryptKeyringVector? then "\n" + test.vector.errorDescription else "", "\n";
-
```

Comment:
is this a sad side effect of printing to stdout being slow which adds to test time?
I kind of like printing every test description

Reply:
Yes, this was from performance testing, both because the printing was skewing the results and because my output was getting lost in the reams of other output.

In general, I'm a fan of tests that run quietly, unless something fails.

Reply:
I find it easy to debug or when i'm trying to create one off test vectors the prints help me make sense of the manifest.
Also, when we eventually publish the test vector runner, we may want to print the test descriptions.

Reply:
> In general, I'm a fan of tests that run quietly, unless something fails.

Fair, I think I can add a flag to the config to easily turn on/off the printing. This way when we publish it is easy to turn it off if needed. I'm ok, removing it for now.